website/client/src/components/ui/filterDropdown.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <span>
     <span class="dropdown-label">{{ label }}</span>
-    <b-dropdown right="right">
+    <b-dropdown
+    :text="(selectedItem.id.includes('sortBy')) ? selectedItem.id.substring(6) : selectedItem.id"
```

Comment:
this doesn't seem to be using the correct string version but simply the key meaning it won't be translated correctly, seeing this instead of the correct string that would be the capitalized "Warrion
<img width="190" alt="Schermata 2019-10-31 alle 12 17 58" src="https://user-images.githubusercontent.com/589911/67942659-9d2d2500-fbd8-11e9-995d-f87d1f2b69ff.png">

I mean that the default selection should match the item listed in the dropdown (with the same text and icon)
"

Reply:
I'm new to vue 
So would you please help me with this
How can I grab the value of the dropdown menu

I found out that "text" attribute returns the same value in the dropdown but it only works for class

Reply:
I think this is something that might have broken with a recent version of bootstrap vue because the slot is there so it might be a breaking change we missed. @negue any idea? The slot exists here https://github.com/HabitRPG/habitica/pull/11481/files#diff-b7d94aa4b827758be8c8eddcd201c4e4R7 but is not rendered

Reply:
@prosarahgamal already found the correct slot name :tada:

@paglias yeah another vue-bootstrap upgrade issue I'll go through other components and see if we still have some removed properties / bindings